### DISCUSSION TITLE
Welcome to win-vind community!

### DISCUSSION THREAD

**Discussion by pit-ray** [pit-ray/win-vind#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by gilbh** [comment]
Hello!

I am a big fan!

When will "e" roll in? I need it very much for textual work. Is there a specific reason why it is difficult to implement?

With thanks!!

**Reply by pit-ray** [reply]
To reproduce the perfect `w` and `e`, semantic analysis is required. To make matters worse, for win-vind to get the text, it has to copy the text once, retrieve it from the clipboard, parse it, and paste it again. Thus, the performance is very poor.

The `~` command implemented in such a way is so loaded that it does not allow to replace continuously by pressing and holding it.

Therefore, if pseudo-simple and simple enough, I can implement it.

**Comment by gilbh** [comment]
I see.

Since `e` is only a caret position command (and not text replacement like `~`) I assume it will not be very slow but behave more like `w` is today (which is great)?

Also, I think partial implementation can still be very useful. For example, I've noticed partial implementation when in Word I move down/up from line to line: vind moves one visual line every time and not according to sentence structure (with line break). But it is not difficult to follow and use it even as it is ...

My point is that even with partial implementation, the usage of `e` could be really great and helpful. Would you consider implementing a partial `e` (like you might be doing with `w`)?

Thanks!

**Reply by pit-ray** [reply]
I understand.   
I moved it to the highest TODO. (ref. https://github.com/pit-ray/win-vind/projects/2#card-60790113)

**Reply by gilbh** [reply]
Thanks!!

**Reply by pit-ray** [reply]
An in-progress release  #76 with the addition of the `e` command is now available.  
Please try it out and give us feedback.

**Comment by gilbh** [comment]
You may wish to add a post to this reddit thread:
[Is there a plugin to use VIM mode in Microsoft Word?](https://www.reddit.com/r/vim/comments/i4gpz0/is_there_a_plugin_to_use_vim_mode_in_microsoft/)

**Reply by pit-ray** [reply]
Since it was over a year ago, I will not mention it. However, I welcome the expansion of the community.

**Comment by gilbh** [comment]
Unrelated question: can I map the print screen key `prt sc` and what is it called as `<>` tag?

**Reply by pit-ray** [reply]
The print screen key is the `<snapshot>` keyword.
See the bottom of https://pit-ray.github.io/win-vind/cheat_sheet/keywords/.


What is <>tag? Please details.

**Comment by gilbh** [comment]
Thank you! I was just looking for the `<snapshot>` and called it a `<>tag`. Nothing meant by that.

**Reply by pit-ray** [reply]
I see. Thanks!

**Comment by Dawid-Deszcz** [comment]
Hi guys I just installed win-vind. This is exactly what I was looking for. Thank you for all your hard work and let me know if I can be of help for you
